### ISSUE TITLE
[server] allow --cwd without any other action

### ISSUE THREAD

**Issue by Simn** [HaxeFoundation/haxe#8842]
The compilation server should consider a `--cwd dir` to be a valid standalone command to put the server into a different directory. At the moment it applies the directory change, then prints the CLI usage and exits.

**Comment by RealyUniqueName** [comment]
What's the purpose of such a command? To omit it on further requests?

**Comment by RealyUniqueName** [comment]
I think there should be a separate arg for that (e.g. --chdir), because --cwd is not defined to behave like that.

**Comment by Simn** [comment]
What do you mean? It already behaves like that, the only issue is that the compiler thinks nothing has happened and prints the default usage.

**Comment by nadako** [comment]
Does the changed directory persist between requests when `--cwd` was used with some other command?

**Comment by RealyUniqueName** [comment]
Yeah, I mean `--cwd` does not change current working dir permanently. It only affects current request iirc

**Comment by Simn** [comment]
I don't think that's true

**Comment by RealyUniqueName** [comment]
Ok then

**Comment by Simn** [comment]
Arguably it should be true though, in which case I'd agree with you...

**Comment by Simn** [comment]
As @back2dos points out, there's a problem doing this because the `--cwd dir` might now end up in `--each`.
